src/runtime/typemanager.cs

```diff
@@ -617,7 +625,7 @@ int MAP_ANONYMOUS
                         case OperatingSystemType.Linux:
                             return 0x20;
                         default:
-                            throw new NotImplementedException($"mmap is not supported on {Runtime.OperatingSystemName}");
+                            throw new NotImplementedException($"mmap is not supported on this operating system");
```

Comment:
NIT: here and below you can pass the `Runtime.OperatingSystem`, since that is the value used in `switch`